File: 백준/Bronze/2738. 행렬 덧셈/행렬 덧셈.py
a, b = [], []

n, m = map(int, input().split())

for row in range(n):
    row = list(map(int, input().split()))
    a.append(row)

for row in range(n):
    row = list(map(int, input().split()))
    b.append(row)


for row in range(n):
    for column in range(m):
        print(a[row][column] + b[row][column],end=' ')
    print()

File: 백준/Bronze/2738. 행렬 덧셈/행렬 덧셈.java
import java.io.*;

public class Main {
    public static void main(String[] args) throws IOException {
        InputStreamReader reader = new InputStreamReader(System.in);
        BufferedReader br = new BufferedReader(reader);
        StringBuilder sb = new StringBuilder();
        
        String[] NM = br.readLine().split(" ");
        int N = Integer.parseInt(NM[0]);
        int M = Integer.parseInt(NM[1]);
        
        int[][] matrix = new int[N][M];
        
        for(int n=0; n<N; n++) {
            String[] row = br.readLine().split(" ");
            for(int m=0; m<M; m++) {
                matrix[n][m] = Integer.parseInt(row[m]);
            }
        }
        
        for(int n=0; n<N; n++) {
            String[] row = br.readLine().split(" ");
            for(int m=0; m<M; m++) {
                matrix[n][m] += Integer.parseInt(row[m]);
            }
        }
        
        for(int n=0; n<N; n++) {
            for(int m=0; m<M; m++) {
                sb.append(matrix[n][m]).append(" ");
            }
            sb.append("\n");
        }
        
        System.out.println(sb);
    }
}